ui/page/wallets/sign_message_page.go

```diff
@@ -230,6 +229,7 @@ func (pg *SignMessagePage) updateButtonColors() {
 func (pg *SignMessagePage) Handle() {
 	gtx := pg.gtx
 	pg.updateButtonColors()
+	pg.Load.EnableKeyEvent = true
```

Comment:
Any reason this was done?

Reply:
Yeah,
I moved it here because when you open the info modal on a page with editors (the tab event is present here) and the modal is closed, tab key does not switch editors anymore because it was initialised OnResume